spring-boot/src/test/java/packages/informationmanipulation/AddUserTest.java

```diff
@@ -8,13 +9,20 @@
 import java.util.HashMap;
 
 class AddUserTest {
+    ReturnUserInformation r;
```

Comment:
Change variable

Reply:
There are a bunch more occasions where this happens, not sure if you want to change but I'd recommend doing so.